tests/wpt/web-platform-tests/2dcontext/building-paths/canvas_complexshapes_arcto_001-ref.htm

```diff
@@ -4,6 +4,7 @@
         <title>HTML5 Canvas Test:  arcTo() adds to subpath if same point</title>
         <link rel="author" title="Microsoft" href="http://www.microsoft.com" />
     </head>
+    lol
```

Comment:
lol?

Reply:
It's going to be reverted next time I do a real sync from upstream.